### PR TITLE
[stable/nats] Configure number of connect retries for implicit routes

### PR DESCRIPTION
**Pull request by devKlausS**
Signed-off-by: devKlausS <schaeffer@linkbit.io>

#### Is this a new chart
NO

#### What this PR does / why we need it:
added possibility to configure number of connect retries for implicit routes. see pull request of nats server https://github.com/nats-io/nats-server/pull/409

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

@bitnami-bot

### CODE REVIEW

File: stable/nats/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nats
-version: 4.2.12
+version: 4.2.13
```
Comment (carrodher) [inline stable/nats/Chart.yaml]: As it is a new feature we usually bump the chart version in a minor.
```suggestion
version: 4.3.0
```

File: stable/nats/values.yaml [hunk 2]
```diff
@@ -218,6 +222,7 @@ cluster:
     ##
     # loadBalancerIP:
 
+
```
Comment (carrodher) [inline stable/nats/values.yaml]: This line appears

**Review by carrodher — CHANGES_REQUESTED** [review]
Thank you very much for your contribution, I just added some small comments.

File: stable/nats/values.yaml (hunk 2 above)
Comment (devKlausS) [reply stable/nats/values.yaml]: fixed in https://github.com/helm/charts/pull/20642/commits/4c1053a9da759afc882d8cc7d709bc47b732171a

**Comment by carrodher** [comment]
🤦‍♂ #20651 was merged by the CI/CD some minutes ago, it seems you need to rebase from master before merge this one

**Review by carrodher — APPROVED** [review]

**Comment by devKlausS** [comment]
> 🤦‍♂ #20651 was merged by the CI/CD some minutes ago, it seems you need to rebase from master before merge this one

done

**Pull request merged**